### PR TITLE
Create test.md

### PR DESCRIPTION
**Pull request by VoteWizard**

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+```json
+{
+    "A.1.5.8": {
+        "Name": "Aligned Voter Committee misalignment risk management",
+        "Version": 1,
+        "Type": "Section",
+        "Components": {
+            "Content": "AVC must limit their participation in Maker Governance to the creation of Aligned Governance Strategy and Aligned Scope Proposals that function resiliently based on open, objective processes and stay at a high enough level to ensure micromanagement isn't possible. AVCs must not attempt to create biased or unfair conditions for specific Ecosystem Actors, or attempt to put in place unjustified, granular decisions directly, or indirectly through biased processes. GOV5 must specify processes in which AVCs are deactivated and their members derecognized in case there is clear misalignment risk. Any attempt by AVC to micromanage governance significantly increases the risk of misalignment as there could easily be a hidden conflict of interest behind the attempt at micromanagement. This section must detail principles and processes for detecting and reacting to the edge case risks related to AVCs overstepping their roles in the governance process."
```
Comment (VoteWizard) [inline test.md:13]:
```suggestion
            "Content": "AVC must limit their participation in Maker Governance to the creation of Aligned Governance Strategy and Aligned Scope Proposals that function resiliently based on open, objective processes and stay at a high enough level to ensure micromanagement isn't possible. AVCs must not attempt to create biased or unfair conditions for specific Ecosystem Actors, or attempt to put in place unjustified, granular decisions directly, or indirectly through biased processes. GOV5 must specify processes in which AVCs are deactivated and their members derecognized in case there is a clear misalignment risk. Any attempt by AVC to micromanage governance significantly increases the risk of misalignment as there could easily be a hidden conflict of interest behind the attempt at micromanagement. This section must detail principles and processes for detecting and reacting to the edge case risks related to AVCs overstepping their roles in the governance process. Test Suggestion 1"
```
